central/policy/service/customresource/custom_resource_test.go

```diff
@@ -0,0 +1,143 @@
+package customresource
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/custom_resource.yaml
+var templateFile string
+
+func TestConvertToCR(t *testing.T) {
+	policy := getTestPolicy()
+	converted, err := GenerateCustomResource(policy)
+	require.NoError(t, err)
+	fmt.Println(converted)
```

Comment:
Should this be removed before merge?

Reply:
I want to keep it for the convenience of debugging